plone/dexterity/content.py

```diff
@@ -354,22 +354,20 @@ def _set__name__(self, value):
     __name__ = property(_get__name__, _set__name__)
 
     def UID(self):
-        """Returns the item's globally unique id."""
+        # Returns the item's globally unique id.
         return IUUID(self)
 
     @security.private
     def notifyModified(self):
-        """Update creators and modification_date.
-
-        This is called from CMFCatalogAware.reindexObject.
-        """
+        # Update creators and modification_date.
```

Comment:
Perhaps it's better to fix the [publishable object requirements](http://docs.zope.org/zope2/zdgbook/ObjectPublishing.html#publishable-object-requirements) – or declare a different security model on the class level.

It's always been very surprising that having a docstring changes object publication status.